lib_eio/net.mli

```diff
@@ -27,7 +27,12 @@ type Exn.err += E of error
 val err : error -> exn
 (** [err e] is [Eio.Exn.create (Net e)] *)
 
-(** IP addresses. *)
+(** IP addresses.
+
+  In order to interoperate with the {{:https://opam.ocaml.org/packages/ipaddr/} ipaddr} library we can use it's octet conversion functions.
+  That is, to convert to an [Eio.Ipaddr] we can use [Ipaddr.to_octets ip |> Eio.Net.Ipaddr.of_raw].
```

Comment:
```suggestion
  That is, to convert to an {! Eio.Net.Ipaddr.t} we can use [Ipaddr.to_octets ip |> Eio.Net.Ipaddr.of_raw].
```

That should link to the `t` in the docs :)) (I think)

Reply:
Ah yes, I'm not too familiar with OCaml documentation syntax. Thanks!